src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const sum = days * 40;
+  let result = sum;
+
+  if (days >= 3) {
+    result = sum - 20;
+  }
+
+  if (days >= 7) {
+    result = sum - 50;
```

Comment:
The logic here overrides the discount for 3 days if the number of days is 7 or more. According to typical discount logic, the discount for 7 days should be cumulative with the 3-day discount. Consider adjusting the logic to apply both discounts when applicable.